requirements/requirements-triton.txt

```diff
@@ -1 +1 @@
-triton>=2.0.0
+triton<2.1.0
```

Comment:
Should we specify both, `>=2.0.0,<2.1.0`?

Reply:
Yes, we need to specify both

Reply:
Updated.